### DISCUSSION TITLE
Suddenly cannot view settings

### DISCUSSION THREAD

**Discussion by buffos** [elgatosf/elgato#146] [Help] [answered]
Today, Dec 10, I suddenly cannot see the settings of the plugin I am authoring. 

Deleted, rebuild. The same.
I have used the streamdeck create command to create the plugin, and in the ui part I am not using any websockets code to handle messaging

It worked fine till late last night. The only thing different is that I don't have internet today

I have the same issue with ScriptDeck plugin.

I cannot believe that internet connectivity can be the issue or can it? I am not doing any web requests and even if I did, why are the settings page not appearing

Software version 6.8.0 21252

**Comment by buffos** [comment]
I have temporarily restored internet connection (through my phone) and both settings ui (property panels) show up correctly.

I really do not get it. How can Internet connectivity affect this? 
This should not happen. Is this a known limitation, bug?

**Comment by tiptronic** [comment, marked as answer]
Hello @buffos 

sorry if this caused confusion. The new SDK is still work in progress and the Property Inspector components are not (yet) fully synced to your local npm.

But you can fix this very, very easily:
- just download the `sdpi-components.js` from [https://cdn.jsdelivr.net/gh/geekyeggo/sdpi-components@v3/dist/sdpi-components.js](https://cdn.jsdelivr.net/gh/geekyeggo/sdpi-components@v3/dist/sdpi-components.js)
- place the Javascript file into the `ui`-directory of your plugin
- and replace this line in your property inspectors `index.html` (or whatever you named it)

`<script src="https://cdn.jsdelivr.net/gh/geekyeggo/sdpi-components@v3/dist/sdpi-components.js"></script>`

with 

`<script src="sdpi-components.js"></script>`

Then everything should work offline as well.

Once the SDK is fully merged, you will not need to do this.

I hope this helps and again sorry, if that caused some trouble on your end

**Reply by buffos** [reply]
How didn't I think of it... I feel stupid....

Thank you
